generation/package.json

```diff
@@ -18,6 +18,7 @@
 	"devDependencies": {
 		"babel-generator": "^6.25.0",
 		"babel-types": "^6.25.0",
+		"download-file-sync": "^1.0.4",
```

Comment:
You're not using 'download' dependency anywhere, am I missing something?

Reply:
fixed that 👍